src/writerai/_base_client.py

```diff
@@ -1791,12 +1806,18 @@ async def post(
         cast_to: Type[ResponseT],
         body: Body | None = None,
         files: RequestFiles | None = None,
+        binary_request: FileTypes | None = None,
```

Comment:
nit: should we call this `content` as well?

Reply:
not entirely sure because content maps to FileContent, but FileTypes is sort of a superset to that (file content + file name + mine type)